TLM/TLM/U/TextureUtil.cs

```diff
@@ -22,19 +24,32 @@ public static class TextureUtil {
                                                  IntVector2 atlasSizeHint) {
             var loadedTextures = new List<Texture2D>(spriteDefs.Length);
             var loadedSpriteNames = new List<string>();
+#if DEBUG
+            bool debugResourceLoading = DebugSwitch.ResourceLoading.Get();
```

Comment:
should this be a property? what if user updates debug swiches mid game?

Reply:
I have new version for this module to be merged with named properties.
It will be merged in a later PR
https://github.com/kvakvs/TMPE/blob/u-speedlimits-tool-4/TLM/TLM/State/ConfigData/DebugSettings.cs
If the user wants to debug texture loading they better set this switch before the loading?